services/postgres/seed.ts

```diff
@@ -7,40 +7,116 @@ async function main() {
     await prisma.ruleType.createMany({
       data: [
         {
-          name: "Allowed Origins",
+          id: "secret-key",
+          name: "secret-key",
+          description:
+            "Allows you to add extra layer of security to avoid misusage by others",
+          validationType: "function",
+          isEditable: false,
+          validationValue: "X",
+          isMultiple: false,
+        },
+        {
+          id: "allowed-origins",
+          name: "allowed-origins",
           description: "Allows you to limit app urls that can make requests",
           isEditable: true,
-          isMultiple: true,
           validationType: "regex",
-          validationValue: `https?://(www\\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()!@:%_+.~#?&/\\//=]*)`,
+          validationValue:
+            "https?:\\/\\/(www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()!@:%_\\+.~#?&\\/\\/=]*)",
+          isMultiple: true,
         },
         {
-          name: "Approved Chains",
+          id: "approved-chains",
+          name: "approved-chains",
           description:
             "Allows you to limit chains that can be access via this App",
           isEditable: true,
+          validationType: "regex",
+          validationValue: ".*-.*",
           isMultiple: true,
-          validationType: "function",
-          validationValue: "Array.isArray(value)",
         },
         {
-          name: "Secret Key",
+          id: "allowed-user-agents",
+          name: "allowed-user-agents",
           description:
-            "Allows you to add extra layer of security to avoid misusage by others",
-          isEditable: false,
-          isMultiple: false,
-          validationType: "function",
-          validationValue: 'typeof value === "string" && value.length === 32',
+            "Allows you to limit type of clients that can use this app",
+          isEditable: true,
+          validationType: "regex",
+          validationValue: ".*", // <-- to be revisited when implementing v2 in future
+          isMultiple: true,
         },
         {
-          name: "Allowed UserAgents",
+          id: "rate-limits",
+          name: "rate-limits",
           description:
-            "Allows you to limit type of clients that can use this app",
+            "Allows you to limit number of requests for this app based on period",
           isEditable: true,
-          isMultiple: true,
           validationType: "regex",
-          validationValue: "*", // <-- @note: this is a placeholder, since I couldnt find a regex for this
+          validationValue: "^d+/Pd+Yd+Md+Wd+Dd+Hd+Md+S$",
```

Comment:
I think we want to make the segments optional. For example "1000/P1D" should be valid. Also, there are two `M` signifiers which is ambiguous on the parsing. I recommend using the ISO8601 spec.